golem_messages/message.py

```diff
@@ -878,6 +895,14 @@ class MessageCannotComputeTask(Message):
         'subtask_id'
     ] + Message.__slots__
 
+    class REASON(enum.Enum):
+        WrongCTD = 'wrong_ctd'
+        WrongKey = 'wrong_key'
+        WrongAddress = 'wrong_address'
+        WrongEnvironment = 'wrong_environment'

```

Comment:
old message used to have env name in it too, is this data now lost?

Reply:
I don't think so. Environment is sent in `ComputTaskDef` by requestor, so he'll now what environment is problematic.